### PR TITLE
Restore loaders

### PR DESCRIPTION
**Pull request by alisman**
I'm not sure how this graphicall, with stacked loaders.  But it's usually too fast to see.  We need to view under slower speed, or bigger study

# Checks
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Airbnb React/JSX Style guide](https://github.com/airbnb/javascript/tree/master/react).
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)

# Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [GifGrabber](http://www.gifgrabber.com/).

# Notify reviewers
Read our [Pull request merging
policy](../CONTRIBUTING.md#pull-request-merging-policy). If you are part of the
cBioPortal organization, notify the approprate team (remove inappropriate):

@cBioPortal/frontend

If you are not part of the cBioPortal organization look at who worked on the
file before you. Please use `git blame <filename>` to determine that
and notify them here:

### CODE REVIEW

File: src/shared/components/featureTitle/FeatureTitle.spec.tsx [hunk 1]
```diff
@@ -1,10 +1,9 @@
-import * as FeatureTitle from './FeatureTitle';
 import React from 'react';
 import { assert } from 'chai';
 import { shallow, mount } from 'enzyme';
 import sinon from 'sinon';
 
-describe('FeatureTitle', () => {
+describe('', () => {
```
Comment (adamabeshouse) [inline src/shared/components/featureTitle/FeatureTitle.spec.tsx]: just delete this file?

File: src/shared/components/loader/Loader.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { assert } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+
+describe('FeatureTitle', () => {
```
Comment (adamabeshouse) [inline src/shared/components/loader/Loader.spec.tsx]: describe('Loader'

File: src/shared/components/loader/Loader.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import Spinner from "react-spinkit";
+import { If, Else, Then } from 'react-if';
+
+export interface ILoader {
+    isLoading:Boolean;
+}
+
+export default class Loader extends React.Component<ILoader, {}> {
```
Comment (adamabeshouse) [inline src/shared/components/loader/Loader.tsx]: `LoadingSpinner`? `Loader` kind of sounds like it's more than a visualization

Comment (onursumer) [inline src/shared/components/loader/Loader.tsx]: or maybe `LoadingIndicator`?

File: src/shared/components/featureTitle/FeatureTitle.spec.tsx (hunk 1 above)
Comment (alisman) [inline src/shared/components/featureTitle/FeatureTitle.spec.tsx]: Actually, for our stats, we should try to keep files, even if we didn't write tests. However, we need to actually import the code for it to work (we don't here).  We can look into other ways to get untested code to register.

File: src/shared/components/loader/Loader.tsx (hunk 3 above)
Comment (alisman) [inline src/shared/components/loader/Loader.tsx]: I like Loading Indicator

**Pull request merged**